esp/services/ws_machine/ws_machineService.cpp

```diff
@@ -1199,6 +1202,12 @@ int Cws_machineEx::runCommand(IEspContext& context, const char* sAddress, const
     catch(IException* e)
     {
         exitCode = e->errorCode();
+        if (exitCode == 255)
```

Comment:
Does it only throw to get here ?  Could it return ok with an exit code of 255 ?

Reply:
If it returns ok, the exit code will be set by CFRunSSH. I haven't found 255 exit code in the existing CFRunSSH. If a 255 exit code is added, the code may have different meaning. We may report it with the new meaning.